commands.go

```diff
@@ -481,10 +481,8 @@ func GetDeltaConfig() (max_deltas int, from_full bool) {
 	if hasOrigin {
 		switch origin {
 		case "LATEST":
-			break
```

Comment:
Are you sure it's OK to remove this?

Reply:
Yes, it's fine. https://play.golang.org/p/e4thhtWzYcw

Reply:
Cool